doc/tutorials/transformer/te_init.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>Maybe add a link to the folder containing the examples building on this here? or ad a list of the examples with links to the seperate notebooks. just to make it a bit easier for users.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Not sure I understood. I think we are done here.</p>

Reply:
<p>i would like to se a table of contents for the users which link the notebooks that build on this one here. i think it would make navigating this tutorial a lot more convenient. if you want to solve this in another ticket that would also be fine but please link me the ticket in question in that case</p>

Reply:
<p>I think, what @MarleneKress79789 means, is that, we have as the last part of this notebook, links to the notebooks that use the transformer-extension, e.g.</p><p> </p><ul><li><a href="doc/tutorials/transformer/model_retrieval.ipynb" target="_blank">Model Retrieval </a></li><li><a href="doc/tutorials/transformer/sequence_classification.ipynb" target="_blank">Sequence Classification</a></li><li>...</li></ul><p> </p><p> </p>